pkg/test/expectations/expectations.go

```diff
@@ -22,6 +22,8 @@ import (
 
 	//nolint:revive,stylecheck
 	. "github.com/onsi/gomega"
+	dto "github.com/prometheus/client_model/go"
```

Comment:
what is `dto`? I might just alias this as `prometheus`

Reply:
`dto` is what that package is import as by the prometheus package itself:

```
package prometheus

import (
        "errors"
        "math"
        "sync/atomic"
        "time"

        dto "github.com/prometheus/client_model/go"
)
```

I'll rename it.